ArduPlane/servos.cpp

```diff
@@ -141,7 +141,7 @@ bool Plane::suppress_throttle(void)
     }
 
 #if HAL_QUADPLANE_ENABLED
-    if (quadplane.is_flying()) {
+    if (quadplane.is_flying_vtol()) {
```

Comment:
i'm concerned this may start the fwd throttle if we failsafe to RTL while on the ground